components/TemplateHeader.js

```diff
@@ -75,6 +117,9 @@ ActivityHeader.propTypes = {
   categoryLabel: PropTypes.string.isRequired,
   milestoneType: ActivityTemplatePropTypes.milestoneSlug.isRequired,
   milestoneLabel: PropTypes.string.isRequired,
+  isMilestone: PropTypes.bool.isRequired,
+  // eslint-disable-next-line react/require-default-props
```

Comment:
It will not show an eslint error if you add a default value for `milestoneGoal` to the `ActivityHeader.defaultProps`. I would also not make `isMilestone` a required prrop, and instead default it to `false`. Ex:
```
TemplateHeader.defaultProps = {
milestoneGoal: null,
isMilestone: false
}
```